staging/src/k8s.io/apiserver/pkg/server/options/api_enablement_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"strings"
+	"testing"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	utilflag "k8s.io/apiserver/pkg/util/flag"
+)
+
+type fakeGroupRegisty struct{}
+
+func (f fakeGroupRegisty) IsGroupRegistered(group string) bool {
```

Comment:
hm, i can't find usage of this function anywhere.
is something missing?

also should be lowercase if not used outside of the package.

Reply:
It is a GroupRegisty interface in https://github.com/kubernetes/kubernetes/blob/09cdfc52872fbf4a6a3bfcad54baf216c1238261/staging/src/k8s.io/apiserver/pkg/server/options/api_enablement.go#L107-L110